examples/flutter_firebaseauth/pubspec.yaml

```diff
@@ -0,0 +1,26 @@
+name: flutter_firebaseauth
+description: Flutter Firebase Auth using bloc library
+version: 1.0.0+1
+
+environment:
+  sdk: ">=2.1.0 <3.0.0"
+
+dependencies:
+  flutter:
+    sdk: flutter
+
+  cupertino_icons: ^0.1.2
+  flutter_bloc: any
+  equatable: ^0.2.0
+
+  google_sign_in: ^4.0.1+1
+  firebase_auth: ^0.8.1+4
+
+dev_dependencies:
+  flutter_test:
+    sdk: flutter
+  mockito: ^4.0.0
+
+flutter:
+
```

Comment:
minor but we can remove the newline here.

Reply:
Done.